frontend/src/features/recruit/icons/RecruitEdit.tsx

```
@@ -0,0 +1,31 @@
+interface IProps extends SVGProps<SVGSVGElement> {}
+
+export const RecruitEditIcon = (props: IProps) => {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="32"
+      height="32"
+      fill="none"
+      viewBox="0 0 32 32"
+      {...props}
+    >
```

Biome (2.3.14)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)